p2p/Cargo.toml

```diff
@@ -56,6 +56,7 @@ p2p-backend-test-suite = { path = "backend-test-suite" }
 p2p-test-utils = { path = "p2p-test-utils" }
 storage-inmemory = { path = "../storage/inmemory" }
 test-utils = { path = "../test-utils" }
+tokio = { workspace = true, default-features = false, features = ["io-util", "macros", "net", "rt", "rt-multi-thread", "sync", "time", "test-util"] }
```

Comment:
Well, at this point... why not just "full"?

Reply:
I'm not sure.
* The `full` does not include `test-util` anyway, so it would still have to be listed explicitly.
* It enables some features we don't use: some extra IO stuff, `parking_lot`, support for spawning processes...

I do get the appeal of brevity but still slightly prefer the explicit list.